manila/share/drivers/netapp/options.py

```diff
@@ -193,7 +193,15 @@
                 help='This option enables the logical space reporting on a '
                      'newly created vserver and locical space accounting '
                      'on newly created volumes on this vserver. ',
-                default=False), ]
+                default=False),
+    cfg.IntOpt('netapp_security_cert_expire_days',
+               min=1,
+               max=3652,
+               default=2190,
```

Comment:
Internally this is okay - but I think upstream it should match the NetApp default of 365 days here